src/locales/fr-FR.json

```diff
@@ -1,149 +1,150 @@
-  "$of": "de {0}",
-  "$singlePageView": "Affichage en mode page simple",
-  "$searchByPageNumber": "Recherche par numéro de page.",
-  "$previous": "Précédent",
-  "$previousImage": "Image précédente",
-  "$previousPage": "Page précédent",
-  "$settings": "Paramètres",
-  "$twoPageView": "Affichage double page",
-  "$goHome": "Retour",
-  "$imageUnavailable": "Image non disponible",
-  "$rotateRight": "Pivoter à droite",
-  "$zoomIn": "Zoomer",
-  "$zoomOut": "Dézoomer",
-  "$adjustImage": "Ajuster l'image",
-  "$clearSearch": "Effacer",
-  "$displaying_A_of_B": "{0} {1} de {2}",
-  "$enterKeyword": "Saisir un mot",
-  "$imageLowercase": "image",
-  "$instanceFound": "1 instance de '{0}' trouvée",
-  "$instancesFound": "{0} instances de '{1}' trouvées",
-  "$nextResult": "Résultat suivant",
-  "$pageLowercase": "page",
-  "$previousResult": "Résultat précéde",
-  "$print": "Imprimer",
-  "$resultFoundFor": "résultat trouvé pour ",
-  "$resultsFoundFor": "résultats trouvés pour",
-  "$searchWithinItem": "Chercher dans ce document :",
-  "$locale": "Langues",
-  "$navigatorEnabled": "Navigateur activé",
-  "$clickToZoomEnabled": "Cliquer pour zoomer",
-  "$reducedMotion": "Réduire le mouvement (désactive les animations)",
-  "$preserveViewport": "Conserver le zoom",
-  "$uvWebsite": "<a href='https://github.com/universalviewer/universalviewer#contributors'>En savoir + sur le Universal Viewer</a>",
-  "$custom": "personnalisée",
-  "$embedInstructions": "Pour afficher ce contenu sur votre site internet, copier-collez le code ci-dessous.",
-  "$height": "Hauteur",
-  "$iiifManifest": "Manifeste IIIF",
-  "$shareInstructions": "Pour partager ce contenu, copier l'URL ci-dessous.",
-  "$size": "Taille :",
-  "$width": "Largeur",
-  "$cancel": "Annuler",
-  "$confirm": "Confirmer",
-  "$viewTerms": "Lire les termes et les conditions d'utilisation",
-  "$login": "Connexion",
-  "$log_in": "Connexion",
-  "$logout": "Déconnexion",
-  "$authCORSError": "Votre navigateur ne supporte pas CORS, veuillez mettre à jour votre navigateur pour afficher ce contenu.",
-  "$authorisationFailedMessage": "Votre tentative de connexion semble avoir échoué. Veuillez réessayer.",
-  "$canvasIndexOutOfRange": "Valeur d'index invalide pour la balise Canvas.",
-  "$fallbackDegradedLabel": "Connexion",
-  "$fallbackDegradedMessage": "Connectez-vous pour visualisez ce contenu en résolution maximale.",
-  "$forbiddenResourceMessage": "Vos droits d'accès ne vous permettent pas de visualiser cette image",
-  "$mediaViewer": "Visionneuse multimédia",
-  "$skipToDownload": "Passer aux téléchargements et aux formats alternatifs",
-  "$pleaseEnterValue": "veuillez saisir une valeur.",
-  "$pleaseLogInToViewAtFullQuality": "Veuillez vous connecter pour voir en pleine qualité.",
-  "$vr": "VR",
-  "$currentTime": "Heure actuelle",
-  "$duration": "Durée",
-  "$mute": "Muet",
-  "$pause": "Pause",
-  "$play": "Jouer"
+    "$collapse": "Masquer les contenus",
```

Comment:
Something is causing the two-space indent to get changed to a four-space indent here -- I'm guessing some setting in your editor. That probably explains why we ran into conflicts in the other locale files earlier. Might be worth rolling back the whitespace changes for clarity.

Reply:
Done, thanks Demian